sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -188,6 +188,9 @@ private[hive] class IsolatedClientLoader(
     (name.startsWith("com.google") && !name.startsWith("com.google.cloud")) ||
     name.startsWith("java.lang.") ||
     name.startsWith("java.net") ||
+    name.startsWith("com.sun.") ||
+    name.startsWith("sun.reflect.") ||
```

Comment:
these two lines above are really broad, is this intentional?

Reply:
Do not add them unless we have to do it.